wheel/src/api.rs

```diff
@@ -14,9 +15,14 @@ use pyo3::prelude::*;
 use pyo3::types::{PyBytes, PyTuple};
 use pyo3::wrap_pyfunction;
 
+fn eval_to_py(err: EvalErr) -> PyErr {
+    // Rarely Used in python bindings.
+    pyo3::exceptions::PyValueError::new_err(err.to_string())
+}
+
 #[pyfunction]
 pub fn serialized_length(program: &[u8]) -> PyResult<u64> {
-    Ok(serialized_length_from_bytes(program)?)
+    serialized_length_from_bytes(program).map_err(eval_to_py)
```

Comment:
did `EvalErr` used to have a conversion to `PyErr`?

Reply:
yes

Reply:
was just to str